app/javascript/article-form/components/Title.jsx

```diff
@@ -1,6 +1,15 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
 import Textarea from 'preact-textarea-autosize';
+// We use this magic Textarea component for title field because it's automatically
```

Comment:
this whole block should be before the line that it pertains to ;-)

Reply:
done